haystack/json-schemas/haystack-pipeline-master.schema.json

```diff
@@ -1776,70 +1773,6 @@
       ],
       "additionalProperties": false
     },
-    "CrawlerComponent": {
```

Comment:
Any idea why the CrawlerComponent is removed here? 🤔 I guess these changes were made by the Update Documentation & Code Style action.

Reply:
No idea. Yes, that was done by the automatic action. @ZanSara do you have any idea what happened here?

Reply:
Yep this is very odd, let me look into this